java/modules/core/src/main/java/org/apache/synapse/config/xml/FaultMediatorFactory.java

```diff
@@ -160,7 +162,13 @@ public Mediator createSpecificMediator(OMElement elem, Properties properties) {
         OMElement node = elem.getFirstChildWithName(NODE_Q);
         if (node != null && node.getText() != null) {
             try {
-                faultMediator.setFaultNode(new URI(node.getText()));
+                @SuppressWarnings("startswith") @StartsWith({"https", "file"}) String uri =
+                                                                               node.getText();
+                //TRUE POSITIVE:The child is picked up from the document by matching the name. Although the child
+                //should be a valid URI but it is not guaranteed to have an accepted protocol. More information can be
+                //found about the document on
+                // https://ws.apache.org/axiom/apidocs/org/apache/axiom/om/OMContainer.html#getFirstChildWithName-javax.xml.namespace.QName-
```

Comment:
I didn't find this document very useful for understanding what's happening. Rather, it seems like the source is the `NODE_Q` field of this class, which is from the "node" key in the configuration syntax at the top of the file.